setup.cfg

```diff
@@ -1,2 +1,32 @@
-[bdist_wheel]
-universal=0
+[metadata]
```

Comment:
Might as well put this config in pyproject.toml?

Reply:
I don't have python 3.5 installed, but I suspect it won't work with it. I'll try getting py3.5 and check it though.

Reply:
We can drop 3.5, 3.6 and 3.7 support

Reply:
@hauntsaninja great, that might help. I'll send a PR to do this first.